src/ui/profile_preferences.ui

```diff
@@ -934,7 +934,7 @@
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBox_USE_IRE_DRIVER_BUGFIX">
             <property name="toolTip">
-             <string>&lt;p&gt;Some MUDs (notably all IRE ones) suffer from a bug where they don't properly communicate with the client on where a newline should be. Enable this to fix text from getting appended to the previous prompt line.&lt;/p&gt;</string>
+             <string>&lt;p&gt;Some games (notably all IRE ones) suffer from a bug where they don't properly communicate with the client on where a newline should be. Enable this to fix text from getting appended to the previous prompt line.&lt;/p&gt;</string>
```

Comment:
:thought_balloon: Since IRE does only do MUDs I'd suggest changing the `(notably all IRE ones)` to `(notably all IRE MUDs)`...?

Reply:
They are still games, though.

Reply:
Fair enough, I'll include it